pages/blog/_slug.vue

```diff
@@ -150,16 +154,42 @@ export default {
   content: {
     liveEdit: false,
   },
-  async asyncData({ $content, params }) {
-    const article = await $content('blog', params.slug).fetch()
+  async asyncData({ params }) {
+    const response = await axios.get(`${process.env.strapiUrl}/api/blogs`, {
+      params: {
+        'filters[slug][$eq]': params.slug,
+        populate: '*',
+      },
+    })
```

Comment:
Since here, we are making a network call, a `try-catch`  block might help.

Reply:
good point, though what are you going to do after catching ?

Reply:
Maybe we can redirect to blog index page.